packages/client/src/hooks/useFullScreen.tsx

```diff
@@ -0,0 +1,28 @@
+import { type MutableRefObject, useCallback, useEffect } from 'react'
+import { toggleFullScreen } from '../features/toggleFullScreen/toggleFullScreen'
+import type { KeyboardKeys } from '../env'
+
+export const useFullScreen = <T extends HTMLElement | null>(
+  ref: MutableRefObject<T>,
+  activeKey: KeyboardKeys = `F`
+) => {
+  const onKeyDown = useCallback(
+    (evt: KeyboardEvent) => {
+      const key = evt.code
+
+      if (key === `Key${activeKey}`) {
+        if (ref.current) {
+          toggleFullScreen(ref.current)
+        }
+      }
+    },
+    [ref, activeKey]
+  )
+
+  useEffect(() => {
+    document.addEventListener(`keydown`, onKeyDown)
+    return () => {
+      document.removeEventListener(`keydown`, onKeyDown)
+    }
+  })
```

Comment:
Согласен, хочу добавить в зависимости `[ref.current, activeKey]`, но на такую запись линтер выдает warning: `React Hook useEffect has a missing dependency: 'onKeyDown'. Either include it or remove the dependency array. Mutable values like 'ref.current' aren't valid dependencies because mutating them doesn't re-render the component.eslint[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)`, т.е. предлагается `[ref, activeKey, onKeyDown]`. Подскажи, пожалуйста, какой вариант тут верный?